packages/toolpad-core/src/AppProvider/AppProvider.test.tsx

```diff
@@ -3,18 +3,33 @@
  */
 
 import * as React from 'react';
-import { describe, test, expect } from 'vitest';
+import { describe, test, expect, vi } from 'vitest';
 import { render, screen } from '@testing-library/react';
 import { createTheme } from '@mui/material/styles';
 import { AppProvider } from './AppProvider';
 
+vi.mock('@mui/material/InitColorSchemeScript', () => ({
+  default: (props: any) => {
+    return <script nonce={props.nonce} data-testid="init-color-scheme-script" />;
+  },
+}));
+
 describe('AppProvider', () => {
   test('renders content correctly', async () => {
     render(<AppProvider>Hello world</AppProvider>);
 
     expect(screen.getByText('Hello world')).toBeTruthy();
   });
 
+  test('renders nonce correctly on color scheme script', async () => {
+    const nonce = Buffer.from(crypto.randomUUID()).toString('base64');
```

Comment:
Oh and the `Buffer` here seems to be making `test:browser` fail.

Reply:
Updated the test to use btoa.  Both `test` and `test:browser` are passing now